### DISCUSSION TITLE
Send line to Terminal

### DISCUSSION THREAD

**Discussion by lounkine** [posit-dev/positron#5246] [Feedback] [answered]
In RStudio, a very handy feature is that when I open an .sh file and press Ctrl-enter (I use git bash on windows as my terminal), the line gets executed if the terminal window is open, similar to how an R line is executed in the console.

I missed that feature in Positron.

**Comment by petetronic** [comment, marked as answer]
Hi @lounkine, thanks for trying out Positron! Perhaps we could attach a keybinding to a command like `Run Selected Text In Active Terminal`, with a condition that the editor lang id is shellscript. I will open an enhancement issue from this discussion to capture this request.

**Reply by petetronic** [reply]
Issue tracked here https://github.com/posit-dev/positron/issues/5251
